tests/tests_app/runners/test_cloud.py

```diff
@@ -122,6 +122,48 @@ def test_new_instance_on_different_cluster_fails(self, monkeypatch):
             body=V1ProjectClusterBinding(cluster_id=new_cluster, project_id="default-project-id"),
         )
 
+    @mock.patch("lightning_app.runners.backends.cloud.LightningClient", mock.MagicMock())
+    def test_run_with_default_flow_compute_config(self, monkeypatch):
```

Comment:
We don't need to duplicate the test like this, right? 

I suggest to either have a simple test like this:
```py
def test_app_default_flow_cloud_compute():
    assert LightningApp(Flow()).flow_cloud_compute.name == "flow-lite"
```

or parametrize the existing test to run both cases.

Reply:
@awaelchli  thank you for your comment,
it seems, the simple test (like below) will not be a complete equivalent to the original one, because
```
    def test_app_default_flow_cloud_compute(self):
        dummy_flow = mock.MagicMock()
        monkeypatch.setattr(dummy_flow, "run", lambda *args, **kwargs: None)
        assert LightningApp(dummy_flow).flow_cloud_compute.name == "flow-lite"
```
checks a bit different things: the aim here is to check the requested `CloudCompute` in cloud runtime, not during the app objects creation (similar, but not equal), although having this test as an additional one can be a good thing!

Parametrization of both of those tests seems to be a great idea, just required some small revising of an existing test :)